nixos/modules/security/pam.nix

```diff
@@ -1184,6 +1184,19 @@ in
       };
     };
 
+    security.pam.unixAuthArgs = mkOption {
+      default = "sha512";
+      type = types.str;
+      description = lib.mdDoc ''
+        Arguments to the pam_unix module.
+
+        Consider switching to "yescrypt rounds=10" for
```

Comment:
I could easily find sources that recommend yescrypt over sha512 on the internet but none of those mentioned that you should use 10 rounds. I would leave that out of the recommendation for now.

Reply:
The recommendation is specific to people protecting other on-disk secrets with the same password. For these, the default number of rounds (5) isn't good enough. Here's one blog post that discusses this: https://medium.com/@slimm609/cost-based-password-hashing-b383bbb355df